src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => (
+      [...goods]
```

Comment:
you don't need to make copy array, because each time you click on the button, you will receive a new array, so in fact sorting will not change the original array

Reply:
I agree with @batstolya. We don't need to make the copy